modules/local/add_varcallername_to_bed.nf

```diff
@@ -0,0 +1,47 @@
+process ADD_VARCALLER_TO_BED {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/tabix:1.11--hdfd78af_0' :
+        'biocontainers/tabix:1.11--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(chromsizes)
+
+    output:
+    tuple val(meta), path("*.gz"), path("*.tbi"), optional: true, emit: gz_tbi
```

Comment:
```suggestion
    tuple val(meta), path("*.gz"), path("*.tbi"), emit: gz_tbi
```
Does it need to be optional?

Reply:
I guess not since tbi index will always be generated alongside the bed file